use_cases/gretel.json

```diff
@@ -47,35 +37,77 @@
       }
     },
     {
-      "gtmId": "use-case-timeseries-dgan",
-      "title": "Generate high-quality time series data",
-      "description": "Use DoppelGANger to create accurate and correlated time-series data.",
+      "gtmId": "use-case-natural-lang-gpt",
+      "title": "Generate natural language text using GPT",
+      "description": "Quickly create new labeled examples for natural language tasks.",
       "cardType": "Console",
-      "imageName": "timeseries.png",
-      "modelType": "timeseries_dgan",
+      "imageName": "natural-lang-gpt.png",
+      "modelType": "gpt",
```

Comment:
I think tests are failing on this as it's not a valid model slug

Reply:
That's correct, we have `gpt_x` in the list of valid options -- that's the closest option that Console has. If `gpt` is the correct one (or they're both good), you can update it here https://github.com/gretelai/gretel-blueprints/blob/develop/test_use_cases.py#L27

Reply:
updated to `gpt_x`